resources/ts/Components/Navigation.tsx

```diff
@@ -63,7 +63,7 @@ export default function Navigation({
     <section className={classes} {...props}>
       {children && <div className="fixed top-10 right-16">{children}</div>}
       <Link href="/">
-        <PubliqLogo />
+        <PubliqLogo color="#00aeef" width={50} height={50} />
```

Comment:
You could use this color from our tailwind theme. But this will require us to use className inside of `PubliqLogo`, I think.